### DISCUSSION TITLE
sync command doesn't support access key with "/" in it

### DISCUSSION THREAD

**Discussion by iKrushYou** [juicedata/juicefs#2605] [Q&A] [answered]
using the latest version of juicefs, I tried running `juicefs sync` with an access key that happens to contain a slash.

Here is an example

```
juicefs sync "s3://access-key:secret-key-with/slash@https://s3.us-west-2.amazonaws.com/bucket-src" "s3://access-key:secret-key-with/slash@https://s3.us-west-2.amazonaws.com/bucket-dest"
```

Error being thrown at https://github.com/juicedata/juicefs/blob/a6d7faee661b2eae6f077225b1fdeef7ba6ffbdb/cmd/sync.go#L240

**Comment by zhijian-pro** [comment, marked as answer]
use `%2F` replace  `/`
